app/models/ephemera_field.rb

```diff
@@ -1,7 +1,10 @@
 # frozen_string_literal: true
+# Class for EphemeraFields as independent resources
+# Please note that these are global in namespace and distinct from ControlledVocabulary::EphemeraField
+# Also note that :field_name was used (as opposed to :name) for the purposes of disambiguation
```

Comment:
It's actually the `name` part of this variable that I find misleading. Since it's effectively a number, but `name` to me implies a word. maybe `key` or `field_key` would be better.